tests/manage/cluster/nodes/test_disk_failures.py

```diff
@@ -49,7 +61,7 @@ def test_detach_attach_worker_volume(self, aws_obj, resources):
         aws_obj.detach_volume(ec2_volume)
 
         # Validate cluster is still functional
-        sanity_helpers.create_resources(resources=resources)
+        self.sanity_helpers.create_resources(pod_factory)
```

Comment:
this is confusing,
pod_factory will create a resource for you and here you are re-creating it?

Reply:
sending pod_factory to create_resources() so the pods will be created inside create_resources() using pod_factory.
I can change the name of create_resources()